File: challenges/4-fizz-buzz/cpp/raffreitas/fizz-buzz.cpp
#include <bits/stdc++.h>

using namespace std;

void fizzBuzz(int n) {

  cout << "saida: " << endl;
  for (int i = 1; i <= n; i++) {
    if (i % 3 == 0 && i % 5 == 0)
      cout << "FizzBuzz" << endl;
    else if (i % 3 == 0 && i % 5 != 0)
      cout << "Fizz" << endl;
    else if (i % 5 == 0 && i % 3 != 0)
      cout << "Buzz" << endl;
    else
      cout << i << endl;
  }
}

int main() {

  int n;

  cin >> n;
  fizzBuzz(n);

  return 0;
}

File: challenges/4-fizz-buzz/java/herco/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Entre com um numero: ");
        int numero = scanner.nextInt();
        
        scanner.close();

        System.out.println("Saida: ");
        for(int i = 1; i <= numero; i++){
            if(i % 3 == 0 && i % 5 == 0) System.out.println("FizzBuzz");
            else if(i % 3 == 0) System.out.println("Fizz");
            else if(i % 5 == 0) System.out.println("Buzz");
            else System.out.println(i);
        }
    }
}
